### DISCUSSION TITLE
I have made some nice and simple GUI for your fast whisper so if someone want a fast click solution it can check it.

### DISCUSSION THREAD

**Discussion by search620** [SYSTRAN/faster-whisper#92] [Show and tell]
This is a one python script 
[Fast_Whisper_GUI.zip](https://github.com/guillaumekln/faster-whisper/files/11106141/Fast_Whisper_GUI.zip)

![image](https://user-images.githubusercontent.com/56544202/228683778-221d0b18-fc29-4c72-81a0-384ee102b2de.png)

it have some log info. multi input files. total duration of the file's, cpu&gpu usage, and such.
Now in my end it is working very nice but it may need nvidia card or another dependencies.
also it is possible to uncheck the gpu box for using just the cpu.

Now it is important to say that I have no experience in building a GUI or even writing code.
But I was helped by a friend and also by artificial intelligence.

I would love to see someone take it and make it even better.
So if someone like me wanted a GUI for this repo to make the life a little easier. Feel free to try it and I hope it will work for you. enjoy.

By the way I am using nvidia gpu so I hope it will work also on your side.

Love and peace with peach

**Comment by sumitmitra255** [comment]
Nice  i will use it and make some adjustments
